app/scripts/proactive/view/LoginView.js

```diff
@@ -22,10 +22,7 @@ define(
 
             fill: function() {
                 var username = this.getCookie('username');
-                if (username == "null") {
-                    $("#user").val("");
-                } else {
-
+                if (username != "null") {
```

Comment:
shoudn't be null without double quotes?

Reply:
If the cookie variable is not defined, it has this value "null", not null, or ""
So the check has to be done on this exact value "null"

Reply:
@marcocast this was the same remark by @Aminelouati :)